core/payment/src/api.rs

```diff
@@ -5,22 +5,27 @@ use ya_client_model::payment::PAYMENT_API_PATH;
 use ya_persistence::executor::DbExecutor;
 use ya_service_api_web::scope::ExtendableScope;
 
-mod provider;
-mod requestor;
+mod accounts;
+mod allocations;
+mod debit_notes;
+mod invoices;
+mod payments;
 
-pub fn provider_scope() -> Scope {
-    Scope::new("/provider").extend(provider::register_endpoints)
-}
-
-pub fn requestor_scope() -> Scope {
-    Scope::new("/requestor").extend(requestor::register_endpoints)
+pub fn api_scope(scope: Scope) -> Scope {
+    scope
+        .extend(accounts::register_endpoints)
+        .extend(allocations::register_endpoints)
+        .extend(debit_notes::register_endpoints)
+        .extend(invoices::register_endpoints)
+        .extend(payments::register_endpoints)
 }
 
 pub fn web_scope(db: &DbExecutor) -> Scope {
     Scope::new(PAYMENT_API_PATH)
         .data(db.clone())
-        .service(provider_scope())
-        .service(requestor_scope())
+        .service(api_scope(Scope::new("")))
+    // TODO: TEST
+    // Scope::new(PAYMENT_API_PATH).extend(api_scope).data(db.clone())
```

Comment:
both defaults below should be 5 seconds according to actual spec:
- pollTmeout: https://github.com/golemfactory/ya-client/blob/87f106498a7a7b1519d82d0981fc21b260a89cf2/specs/common.yaml#L18
- ackTimeout: https://github.com/golemfactory/ya-client/blob/87f106498a7a7b1519d82d0981fc21b260a89cf2/specs/common.yaml#L32

Reply:
referenced in #919